CITATION.cff

```diff
@@ -18,16 +18,16 @@ authors:
   orcid: https://orcid.org/0000-0003-2354-1988
```

Comment:
> Maybe we should also move Wei Ji (643) before Leo (258)?

I'm ok with moving up the list, but I'm thinking if we should move Leo to second last (before Paul)? At least in my field, supervisory roles are placed in the last author(s) position, but unsure about what others think here.

Oh, and if moving Leo, we might as well update his affiliation to [Universidade de São Paulo](https://www.iag.usp.br/).

Reply:
What about keeping the author order in this release and having more discussions later, since the release due date is very close?